api/checks/update_test.go

```diff
@@ -0,0 +1,76 @@
+// +build small
+
+package checks
+
+import (
+	"context"
+	"net/url"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/web-platform-tests/wpt.fyi/api/checks/summaries"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+	"github.com/web-platform-tests/wpt.fyi/shared/sharedtest"
+)
+
+func TestGetDiffSummary_Regressed(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	before := shared.TestRun{}
```

Comment:
Can you add helper functions for some of the common code? Might make it easier to tell what's changing between tests. AFAICT it's just the TestDiff and the summary lines, right?